packages/legacy/utils/src/object.ts

```diff
@@ -153,3 +153,25 @@ export const fromEntries = <T extends unknown>(entries: [string, any][]) =>
  */
 export const getCSSVar = (theme: Dict, scale: string, value: any) =>
   theme.__cssMap?.[`${scale}.${value}`]?.varRef ?? value
+
+/**
+ * Object.assign which moves properties to end of object rather than replacing them
+ * */
+export const assignAfter = (target: object, ...sources: any[]) => {
```

Comment:
I think you meant to stick this in `packages/utilities/object-utils` and not `packages/legacy/utils`? 😅